src/gui/dialogs/editor/custom_tod.cpp

```diff
@@ -251,8 +294,7 @@ void custom_tod::update_tod_display()
 	// The display handles invaliding whatever tiles need invalidating.
 	disp->update_tod(&get_selected_tod());
 
-	// NOTE: revert to invalidate_layout if necessary to display the ToD mask image.
-	get_window()->queue_redraw();
+	get_window()->invalidate_layout();
```

Comment:
Why this change?

Reply:
When there is no ToD selected, if the user selects an image using the browse button, the image won't be reloaded unless `invalidate_layout` is used.